gp-includes/things/translation.php

```diff
@@ -249,9 +249,21 @@ public function normalize_fields( $args ) {
 
 	public function prepare_fields_for_save( $args ) {
 		$args = parent::prepare_fields_for_save( $args );
+
+		/**
+		 * Filters the translation fields before they are saved.
+		 *
+		 * @since 3.0.0
+		 *
+		 * @param array          $args Translation arguments.
+		 * @param GP_Translation $this Translation class.
+		 */
+		$args = apply_filters( 'gp_translation_prepare_for_save', $args, $this );
+
 		if ( is_array( gp_array_get( $args, 'warnings' ) ) ) {
 			$args['warnings'] = serialize( $args['warnings'] );
 		}
+		
```

Comment:
> gp-includes/things/translation.php:266:1: error - Whitespace found at end of line (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)

Please remove the space and rebase again.

Reply:
doh, seems a story neverending